tests/test_data_k.py

```diff
@@ -8,12 +8,14 @@
 from wannierberri.__Kpoint import KpointBZ
 from wannierberri.__Data_K import Data_K
 
-from create_system import create_files_Fe_W90, system_Fe_W90
+from create_system import create_files_Fe_W90, system_Fe_W90, system_Fe_W90_wc
 
 
+@pytest.fixture(scope="module")
 def test_fourier(system_Fe_W90):
```

Comment:
If you use `system_Fe_wc`, you need to list it i the brackets `()`

Reply:
yes